.gitignore

```diff
@@ -25,5 +25,6 @@ lerna-debug.log
 # example ignores
 examples/contracts/**/out
 
-docs/.vitepress/dist
-docs/.vitepress/cache
\ No newline at end of file
+site/.vitepress/dist
+site/.vitepress/cache
+site/**/*.js
```

Comment:
does this ignore the snippets? any reason this was added?

Reply:
it does not ignore snippets. I did not add these, just fixed `docs` to `site` because thats the name of the `site` package folder.